cogboard-app/src/main/resources/initData/config.json

```diff
@@ -8,6 +8,7 @@
         "title": "Welcome to Cogboard",
         "columns": 8,
         "theme": "default",
+        "type": "WidgetBoard",
```

Comment:
I think this change should be optional - in order to be backwards compatible we should have fallback to `WidgetBoard` if this is non existing.

Reply:
Yes, good suggestion. This is the part that I have mentioned yesterday and will be working on today - to provide some backward compatibility.

Reply:
Done